tests/bindings-test/dotnet/shared.mk

```diff
@@ -0,0 +1,29 @@
+TOP=../../../..
+include $(TOP)/Make.config
+include $(TOP)/mk/colors.mk
+
+prepare:
+	$(Q) $(MAKE) -C $(TOP)/tests/dotnet copy-dotnet-config
+
+reload:
+	$(Q) rm -Rf $(TOP)/tests/dotnet/packages
```

Comment:
does that still exists ? it did not for other projects

Reply:
No, it doesn't, I just copied this file from elsewhere. Let me remove it.